tasks/main.yml

```diff
@@ -129,6 +160,18 @@
 
 - name: Create Consul configuration
   include: config.yml
+  when: ansible_os_family != 'Windows'
+
+- name: Create Consul configuration for Windows
+  include: config_windows.yml
+  when: ansible_os_family == 'Windows'
+
+- name: Ensure neither ACL nor TLS are requested for windows hosts
```

Comment:
Small safety check. I haven't updated the ACL or TLS code, so in the meanwhile their support is turned off for Windows hosts. This can be added later.